content_schemas/formats/fatality_notice.jsonnet

```
@@ -14,6 +14,9 @@
         body: {
           "$ref": "#/definitions/body",
         },
+        casualties: {
+          type: "array",
+        },
```

I think this needs to specify the type of content that is permitted within the array, e.g.
```suggestion
        casualties: {
          type: "array",
          items: {
            type: "string"
          }
        },
```

---

yup, right, done!